### PR TITLE
Many OE style guide-conformance tweaks and other minor edits

### PR DESCRIPTION
**Pull request by cshoredaniel**
iwinfo intermittently fails with BitBake 1.34 due to parallel make issues so disable parallel make for iwinfo.

A whole whack of recipes-core style guide conformance fixes.

And some a minor bug or two.

### CODE REVIEW

**Comment by cshoredaniel** [comment]
Removed the iwinfo parallel make bugfix into a separate PR

**Comment by cshoredaniel** [comment]
Moved procd rc.d bugfix out of this series.

File: recipes-core/fstools/fstools_git.bb [hunk 1]
```diff
@@ -9,17 +9,18 @@ HOMEPAGE = "https://git.openwrt.org/?p=project/fstools.git;a=summary"
 LICENSE = "LGPL-2.1 & GPL-2.0 & PD"
 LIC_FILES_CHKSUM = "file://ubi.c;beginline=1;endline=17;md5=8ccc371d64f0b3a8d91065b678dc7095"
 SECTION = "base"
+DEPENDS += "util-linux"
```
Comment (kraj) [inline recipes-core/fstools/fstools_git.bb]: this seems to be new if this is needed add a reason to commit

Comment (cshoredaniel) [reply recipes-core/fstools/fstools_git.bb]: oops, will fix that.

File: recipes-core/fstools/fstools_git.bb [hunk 2]
```diff
@@ -9,17 +9,18 @@ HOMEPAGE = "https://git.openwrt.org/?p=project/fstools.git;a=summary"
...
 
-SRCREV = "37762ff488752c1686b359816aec75831b49c55f"
 SRC_URI = "git://git.openwrt.org/project/fstools.git;branch=lede-17.01 \
 	   file://0001-Define-GLOB_ONLYDIR-if-not-available.patch \
           "
-DEPENDS += "util-linux"
```
Comment (cshoredaniel) [inline recipes-core/fstools/fstools_git.bb]: Actually it was here.  So not new.

File: recipes-core/fstools/fstools_git.bb (hunk 1 above)
Comment (cshoredaniel) [reply recipes-core/fstools/fstools_git.bb]: as noted below it's not actually new, just moved.

**Pull request merged**